tests/helics/application_api/FilterTests.cpp

```diff
@@ -1147,6 +1304,125 @@ TEST_F(filter_tests, reroute_separate2_5message)
     broker->waitForDisconnect();
 }
 
+TEST_F(filter_tests, reroute_separate2_5000message_ci_skip)
```

Comment:
Does this test take a while to run?

Reply:
not that long,  just didn't see a reason to make it run on the CI tests.  Was trying to get it to fail on my machine and didn't succeed, so this test might be able to be removed if desired.